controller/validations.go

```diff
@@ -103,17 +108,22 @@ type ValidationsReconciler struct {
 
 // Reconcile fetches the VirtualService and prints its name
 func (r *ValidationsReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	log.Debug("[ValidationsReconciler] Started reconciling ")
+	// prepare the logger
+	zl := log.FromContext(ctx)
+	zl = log.AddGroup(zl, log.ValidationLogName)
+	ctx = log.ToContext(ctx, zl)
+
+	zl.Debug().Msg("[ValidationsReconciler] Started reconciling ")
```

Comment:
I'm not sure we want to use this logger in the controllers. controller-runtime heavily uses logr for structured logging. Might be annoying to have two different logging APIs but it does provide some use. See: https://github.com/kiali/kiali/pull/8401/files#diff-293a9533f0f78fd250164b1d5c0c05b78528f6e8f6bdc489a7f501948c227c81R85-R96.

Reply:
I reverted those controller changes